fesvr/htif.h

```diff
@@ -144,7 +144,7 @@ TARGET (RISC-V BINARY) OPTIONS\n\
 {"signature", required_argument, 0, HTIF_LONG_OPTIONS_OPTIND + 2 },     \
 {"chroot",    required_argument, 0, HTIF_LONG_OPTIONS_OPTIND + 3 },     \
 {"payload",   required_argument, 0, HTIF_LONG_OPTIONS_OPTIND + 4 },     \
-{"signature-granularity",    optional_argument, 0, HTIF_LONG_OPTIONS_OPTIND + 5 },     \
+{"signature-granularity",    required_argument, 0, HTIF_LONG_OPTIONS_OPTIND + 5 },     \
```

Comment:
Why?

Reply:
Based on the help text, this flag needs an argument. This fixes the getopt_long parsing to respect that, and provide the usage error text if no argument is provided.
```
      --signature-granularity=VAL           Size of each line in signature.\n\
       +signature-granularity=VAL\n\
```